clock/index.js

```diff
@@ -56,4 +56,7 @@ if (featureToggles.isFeatureEnabled('enableQueueStatsActivityLogs')) {
 }
 
 // Clean up completed jobs
-setInterval(jobCleanup.execute, conf.JOB_REMOVAL_INTERVAL_SECONDS * 1000);
+setInterval(jobCleanup.enqueue, conf.JOB_REMOVAL_INTERVAL);
+
+// Reset the job id counter
+setInterval(jobCleanup.resetCounter, conf.JOB_COUNTER_RESET_INTERVAL);
```

Comment:
I added this as a separate clock process that runs less frequently. Resetting the id counter and deleting jobs every time will cause new jobs to overwrite the completed jobs that were skipped.